cirq-superstaq/cirq_superstaq/job.py

```diff
@@ -201,7 +207,14 @@ def counts(self, timeout_seconds: int = 7200, polling_seconds: float = 1.0) -> D
             time_waited_seconds += polling_seconds
 
         self._check_if_unsuccessful()
-        return self._job["samples"]
+        counts_dictionary = self._job["samples"]
+
+        if qubit_indices:
+            counts_dictionary = gss.superstaq_client.get_counts_on_qubits(
+                counts_dictionary, qubit_indices
+            )
+
+        return counts_dictionary
```

Comment:
imo typing info shouldn't be put into the name of an object.  It's kind of like saying `apple_fruit`.
```suggestion
        counts = self._job["samples"]
        if qubit_indices:
            counts = gss.superstaq_client.get_counts_on_qubits(
                counts, qubit_indices
            )
        return counts
```